batch/sql/estimated-current.sql

```diff
@@ -309,6 +309,22 @@ CREATE TABLE IF NOT EXISTS `job_attributes` (
 ) ENGINE = InnoDB;
 CREATE INDEX job_attributes_key_value ON `job_attributes` (`key`, `value`(256));
 
+CREATE TABLE IF NOT EXISTS `region_ids` (
```

Comment:
I think it would be more consistent with the rest of our schema if this table were just named `regions`.